### PR TITLE
Beabranch

### PR DESCRIPTION
**Pull request by LibbyKnight**

### CODE REVIEW

File: assets/javascript/java2.js [hunk 1]
```diff
@@ -162,15 +162,20 @@ $(document).ready(function () {
 			}
 
 		})
-	}
 
+	
+}
+	trailer();
```
Comment (dzimmerman902) [inline assets/javascript/java2.js]: Don't need this call function, it's already listed below

File: secondPage.html [hunk 2]
```diff
@@ -101,16 +101,23 @@ <h3 id="movieTitle"></h3>
...
+							<input type="text" required pattern="(\d{5}([\-]\d{4})?)" value name="postalcodes_pattern0" id="postalcodes_pattern0" list="postalcodes_pattern0_datalist" placeholder="12345">
 
+							
+
+				
 						</div>
 					</div>
 
-					<button class="btn btn-default navbar-btn" id="submit" data-toggle="modal" data-target="#mymodal">Theater Showtimes</button>
+					<button class="btn btn-default navbar-btn" id="submit" data-toggle="modal" data-target="#mymodal">Theater Showtimes</button> </form>
```
Comment (dzimmerman902) [inline secondPage.html]: I need some proper indentation for lines 108 through 120, not sure what's going on.

Is the form element needed? It wasn't there before.

Comment (AdamJahur) [inline secondPage.html]: yea i think the form is needed

**Comment by AdamJahur** [comment]
code looks good jus take out that first call for the trailer function. i think the form tag is needed

**Comment by Nickpopadopolous** [comment]
Looks fine other than what they said.

File: secondPage.html (hunk 2 above)
Comment (LibbyKnight) [inline secondPage.html]: dont take out the form! its needed there. the trailer function, ill delete it per you guys' suggestion, but i thought that a function needs to be called in order for it to run it.

Comment (AdamJahur) [inline secondPage.html]: yea it needs to be called but u have it called twice we only need it to be called once

Comment (AdamJahur) [inline secondPage.html]: you have it called in lines 168 and 175

**Comment by AdamJahur** [comment]
cool looks better

**Pull request merged**